views/admin.php.inc

```diff
@@ -227,17 +217,23 @@ function tawk_to_render_widget_iframe($base_url, $iframe_url, $widget=array(), $
         jQuery(document).ready(function() {
             if(jQuery("#always_display").prop("checked")){
                 jQuery('.show_specific').prop('disabled', true);
+                $(".div_show_specific").hide();
```

Comment:
Use `jQuery` instead of `$`

Reply:
Good tip.  When I recently learned about jQuery all the guides used the dollar.  When I searched for jQuery or dollar I found reference to the possibility for conflict where another framework (i.e. PHP in this case) uses dollar for variables.  I've made this change.